client/src/component/ServiceCard/style.js

```diff
@@ -3,13 +3,13 @@ import { makeStyles } from '@material-ui/core/styles';
 const useStyles = makeStyles((theme) => ({
   card: {
     height: 400,
+    display: 'flex',
   },
   root: {
-    borderBottom: '1px solid #CFAB7A',
     width: 240,
     textAlign: 'center',
     display: 'flex',
-    flexDirection: 'column',
+    flexDirection: 'column ',
     justifyContent: 'flex-end',
   },
   image: {
```

Comment:
suggestion: 
add an object-fit property to the image to keep the aspect ratio of the image intact
you can use `object-fit: contain` to scale the image down to fit the image container
or you can use `object-fit: cover` to crop the image to fit the image container
[object fit mdn](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit)